modules/tensor_mechanics/src/materials/ComputeMultiPlasticityStress.C

```diff
@@ -137,14 +140,20 @@ ComputeMultiPlasticityStress::computeQpStress()
   bool ld_encountered = false;
   bool constraints_added = false;
 
-  // try a purely elastic step first
-  bool found_solution = elasticStep(_stress_old[_qp], _stress[_qp], _intnl_old[_qp], _intnl[_qp], _plastic_strain_old[_qp], _plastic_strain[_qp], _my_elasticity_tensor, _my_strain_increment, _yf[_qp], number_iterations, _Jacobian_mult[_qp]);
+  // plastic multipliers and cumulative plastic multipliers.
+  // Here they are used by quickStep below and are unimportant dummies
+  std::vector<Real> pm;
+  pm.assign(_num_surfaces, 0);
+  std::vector<Real> cumulative_pm;
+  cumulative_pm.assign(_num_surfaces, 0);
+
+  // try a "quick" return first - this can be purely elastic, or a customised plastic return defined by a TensorMechanicsPlasticXXXX UserObject
+  bool found_solution = quickStep(_stress_old[_qp], _stress[_qp], _intnl_old[_qp], _intnl[_qp], pm, cumulative_pm, _plastic_strain_old[_qp], _plastic_strain[_qp], _my_elasticity_tensor, _my_strain_increment, _yf[_qp], number_iterations, _Jacobian_mult[_qp], "computeQpStress", true);
```

Comment:
No, the additional bool variable helps the readability, but you should make it `const`. It is just a placeholder that is not intended to be changed.